package.json

```diff
@@ -7,51 +7,55 @@
     "type": "git",
     "url": "https://github.com/VivyTeam/krypt-web.git"
   },
-  "main": "dist/krypt-web.min.js",
   "license": "MIT",
-  "files": [
-    "/dist"
-  ],
   "scripts": {
-    "build:package": "rimraf dist && webpack",
+    "build:package": "pack build",
     "deploy:documentation": "jsdoc src/lib && git-directory-deploy --directory out/",
     "linter:check": "eslint src/**/*.js",
-    "test": "karma start --single-run --browsers ChromeHeadless karma.conf.js",
-    "test:watch": "karma start --auto-watch --browsers ChromeHeadless karma.conf.js"
+    "test": "karmatic",
+    "test:watch": "karmatic watch"
+  },
+  "@pika/pack": {
+    "pipeline": [
+      [
+        "@pika/plugin-standard-pkg",
+        {
+          "exclude": [
+            "tests/**/*"
+          ]
+        }
+      ],
+      [
+        "@pika/plugin-build-node"
+      ],
+      [
+        "@pika/plugin-build-web"
+      ],
+      [
+        "@pika/plugin-build-types"
+      ]
+    ]
   },
   "dependencies": {
     "scrypt-async": "2.0.1"
   },
   "devDependencies": {
-    "babel-core": "6.26.3",
-    "babel-eslint": "^8.2.6",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-stage-0": "^6.24.1",
-    "chai": "^4.2.0",
+    "@pika/pack": "^0.3.7",
+    "@pika/plugin-build-node": "^0.3.16",
+    "@pika/plugin-build-types": "^0.3.16",
+    "@pika/plugin-build-web": "^0.3.16",
+    "@pika/plugin-standard-pkg": "^0.3.16",
     "eslint": "^5.3.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-config-prettier": "^3.6.0",
-    "eslint-loader": "^2.1.2",
-    "eslint-plugin-import": "^2.16.0",
-    "eslint-plugin-prettier": "^3.0.1",
+    "eslint-config-vivy": "^1.0.6",
     "git-directory-deploy": "1.5.1",
     "husky": "1.3.1",
     "jsdoc": "^3.6.2",
-    "karma": "^3.1.4",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-mocha": "^1.3.0",
-    "karma-spec-reporter": "^0.0.32",
-    "karma-webpack": "^3.0.5",
-    "mocha": "^5.2.0",
+    "karmatic": "^1.3.1",
     "prettier": "^1.16.1",
     "pretty-quick": "^1.10.0",
     "raw-loader": "1.0.0",
     "rimraf": "^2.6.3",
-    "webpack": "^4.29.0",
-    "webpack-cli": "^3.2.3"
+    "webpack": "^4.32.0"
```

Comment:
🤔 wondering, why we still need webpack?

Reply:
Its a peer dependency of karmatic so is expected to be installed by the consumer, check out the installation instructions here: https://github.com/developit/karmatic